### DISCUSSION TITLE
GPU Hardware Question

### DISCUSSION THREAD

**Discussion by LutzRohlfing** [Exawind/amr-wind#1397] [Q&A] [answered]
Hello Forum,

My research group and I are planning to purchase a powerful desktop workstation for simulations with AMR-Wind and OpenFAST. We are considering an AMD W7900 GPU with 48GB VRAM. Are there any recommendations for GPUs from AMR-Wind's perspective? Is this GPU suitable for computations with AMR-Wind?

Thanks in advance!

**Comment by michaelasprague** [comment, marked as answer]
We aren't aware of anyone using AMD W7900 for AMR-Wind calculations, so we can't comment on performance.  We've had success with, e.g., NVIDIA H100, AMD Instinct 250X, amongst others.
